Include/monitoring.h

```diff
@@ -0,0 +1,276 @@
+#ifndef Py_MONITORING_H
+#define Py_MONITORING_H
+#ifdef __cplusplus
+extern "C" {
+
+#endif
+
+typedef struct _PyMonitoringState {
+    uint8_t active;
+    uint8_t opaque;
+} PyMonitoringState;
+
+
+#ifndef Py_LIMITED_API
+#  define Py_CPYTHON_MONITORING_H
+#  include "cpython/monitoring.h"
+#  undef Py_CPYTHON_MONITORING_H
+
+#define IF_ACTIVE(X)  \
+    if (state->active) { \
+        return (X); \
+    } \
+    else { \
+        return 0; \
+    }
+
+#endif
+
+
+#ifndef Py_LIMITED_API
+static inline void
+PyMonitoringScopeBegin(PyMonitoringState *state_array, uint64_t *version,
+                       uint8_t *event_types, uint32_t length)
+{
+    _PyMonitoringScopeBegin(state_array, version, event_types, length);
+}
```

Comment:
Is there a reason why these aren't called `PyMonitoring_BeginScope`, i.e. with an underscore separator in the name (and a verb+object name), like all the other names?